src/test/beatmaptest.cpp

```diff
@@ -62,6 +62,12 @@ TEST_F(BeatMapTest, Scale) {
 
     pMap->scale(Beats::THREEFOURTHS);
     EXPECT_DOUBLE_EQ(bpm / 2, pMap->getBpm());
+
+    pMap->scale(Beats::FOURTHIRDS);
```

Comment:
Yea it's kind of confusing. Do you know how to run the tests locally? 

Run scons with the ```test=1``` flag. Then you should end up with a mixxx-test binary, so you can run ```./mixxx-test``` to see if the tests pass. If you don't want to, then the pull request checks by AppVeyor and Travis will let us know if the tests pass anyway.